cafebabel/templates/articles/proposals/create.html

```diff
@@ -57,6 +57,20 @@ <h3>Wanna be part of Cafébabel project? That’s the right place</h3>
 
     <button type=submit>Suggest this article</button>
   </form>
-  <button><a href=m&#x61;ilto:{{config.EDITORS_EMAIL_DEFAULT|urlencode|replace('.', '&#46;')|replace('a', '&#x61;')|safe}}>Or send your proposal by email</a></button>
+  <a class="button proposal-email" href=m&#x61;ilto:{{config.EDITORS_EMAIL_DEFAULT|urlencode|replace('.', '&#46;')|replace('a', '&#x61;')|safe}}>Or send your proposal by email</a>
+  <script>
+     const editorEmail = {
```

Comment:
What is the indent size of your JS for the project? It was 2 in the previous file and it's 4 here.